### PR TITLE
footer with dead links

### PR DESCRIPTION
**Pull request by JoshMacSween**
Super simple footer, let me know what you think, will work on setting up proper link_to paths etc

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          </ul>
+
+        </div>
+        </div>
+        </div>
+        </footer>
+
+    <!-- Copyright -->
+    <div class="footer-copyright text-center py-3">© 2019 Copyright:
+      <a href="http://yegrb.com/index.html/"> YEGRB.com</a>
```
Comment (alanvardy) [inline app/views/layouts/_footer.html.erb]: This should be root_path. Never hard code links to your own website :smile:

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      </li>
+      </div>
+
+      <hr class="clearfix w-100 d-md-none">
+
+
+        <div class="col-md-3 mx-auto">
+
+
+          <h5 class="font-weight-bold text-uppercase mt-2 mb-2">Inspire</h5>
```
Comment (alanvardy) [inline app/views/layouts/_footer.html.erb]: We don't need all these extra spaces

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+              <a href="#!">Wiki</a>
+            </li>
+          </ul>
+
+        </div>
+
+        <hr class="clearfix w-100 d-md-none">
+
+
+        <div class="col-md-3 mx-auto">
```
Comment (alanvardy) [inline app/views/layouts/_footer.html.erb]: So much space

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            </li>
+            <li>
+              <a href="#!">Our Story</a>
+            </li>
+          </ul>
+
+        </div>
+        </div>
+        </div>
+        </footer>
```
Comment (alanvardy) [inline app/views/layouts/_footer.html.erb]: Indentation needs to be fixed up. Items within should be tabbed in.

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,71 @@
+<footer class="page-footer font-small indigo">
+  <div class="container text-center text-md-left">
+    <div class="row">
+      <div class="col-md-3 mx-auto">
+      <h5 class="font-weight-bold text-uppercase mt-2 mb-2">Connect</h5>
```
Comment (alanvardy) [inline app/views/layouts/_footer.html.erb]: Loving the bootstrap!

File: app/views/layouts/_footer.html.erb (hunk 4 above)
Comment (alanvardy) [reply app/views/layouts/_footer.html.erb]: Refer to _navbar.html.erb for an example if you need one

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -13,5 +13,6 @@
   <%= render 'layouts/navbar' %>
   <%= render 'layouts/alerts' %>
     <%= yield %>
+  <%= render 'layouts/footer'%>
```
Comment (alanvardy) [inline app/views/layouts/application.html.erb]: Beauty

**Comment by alanvardy** [comment]
Also take a look at how you can make the footer stick to the bottom of the screen

I did it on my wife's website: https://corneliakelinske.vardy.codes/
Find the code in here: https://github.com/alanvardy/corneliakelinske

File: app/views/layouts/_footer.html.erb (hunk 4 above)
Comment (JoshMacSween) [reply app/views/layouts/_footer.html.erb]: Will do!

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (JoshMacSween) [reply app/views/layouts/_footer.html.erb]: Haha, ill be fixing all the links on my next iteration 😂

Comment (alanvardy) [reply app/views/layouts/_footer.html.erb]: :+1:

File: app/views/layouts/_footer.html.erb [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+              <a href="#!">Events</a>
+            </li>
+            <li>
+              <a href="#!">Our Story</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </footer>
```
Comment (alanvardy) [inline app/views/layouts/_footer.html.erb]: You have 2 `</footer>` tags

Comment (JoshMacSween) [reply app/views/layouts/_footer.html.erb]: Whups! 😬

**Pull request merged**